http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/db/RuntimeServiceDbQueriesSpec.scala

```diff
@@ -0,0 +1,124 @@
+package org.broadinstitute.dsde.workbench.leonardo
+package http
+package db
+
+import cats.effect.IO
+import org.broadinstitute.dsde.workbench.leonardo.CommonTestData._
+import org.broadinstitute.dsde.workbench.leonardo.config.Config
+import org.broadinstitute.dsde.workbench.leonardo.db.{
+  labelQuery,
+  LabelResourceType,
+  RuntimeServiceDbQueries,
+  TestComponent
+}
+import org.broadinstitute.dsde.workbench.leonardo.http.api.ListRuntimeResponse2
+import org.scalatest.FlatSpecLike
+import org.scalatest.concurrent.ScalaFutures
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+class RuntimeServiceDbQueriesSpec extends FlatSpecLike with TestComponent with GcsPathUtils with ScalaFutures {
```

Comment:
Added this because there wasn't an existing spec for `RuntimeServiceDbQueries` (just `LeonardoServiceDbQueries`).